frontend/src/less/components/autocomplete.less

```diff
@@ -0,0 +1,123 @@
+.autocomplete {
+    position: absolute;
+    background: white;
+    width: 100%;
+    border: 1px solid #0e2d3b;
+    border-top: 0;
+    padding: 20px;
+    z-index: 10;
+    color: #333;
+    text-align: left;
+    display: none;
+    font-size: 14px;
+
+    ul.topics {
+        a {
+            padding: 10px;
+        }
+    }
+
+    a {
+        color: #333;
+        text-decoration: none;
+        display: block;
+
+        &:hover {
+            background: #eee;
+            text-decoration: none;
+        }
+
+        &.more {
+            color: #4dbbeb;
+            text-decoration: underline;
+            display: inline-block;
+
+            &:hover {
+                text-decoration: none;
+                background: transparent;
+            }
+        }
+    }
+
+
+
+    ul {
+        padding: 0;
+        margin-bottom: 15px;
+        li {
+            list-style: none;
+            width: 100%;
+        }
+    }
+
+    .no-topics {
+        margin-bottom: 20px;
+        padding: 0 10px;
+        font-weight: bold;
+    }
+}
+
+.autocomplete__head {
+    border-top: 1px solid #0e2d3b;
+    padding: 20px 0 10px;
+    font-size: 12px;
+
+    &:first-child {
+        border: 0;
+        padding: 0 0 10px;
+    }
+}
+
+ul.upvote__list {
+
+    li {
+        &:hover {
+            background: #eee;
+            text-decoration: none;
+        }
+    }
+
+    a {
+        padding: 10px;
+
+        &:hover {
+            background: transparent;
+        }
+    }
+
+    .upvote__item {
+        width: 441px;
+        float: left;
+        display: inline-block;
+
+        .votes {
+            font-size: 12px;
+            font-weight: normal;
+
+        }
+    }
+
+    .upvote {
+        width: 60px;
+        display: inline-block;
+        text-align: center;
+
+        a {
+            font-size: 12px;
+            display: inline-block;
+            color: #4dbbeb;
+
+            .glyphicon {
+                font-size: 20px;
+                line-height: 10px;
+                top: 5px;
+            }
+        }
+    }
+}
+
+input.first:focus + .autocomplete.first,
```

Comment:
@amcatko input.first is both too generic and too specific :)

- class "first", "second" etc is unnecessary - the + operator will do the work
- rather give some class to the input saying it has autocomplete, like input.with-autocomplete + .autocomplete